### PR TITLE
Improve flow coverage

### PR DESCRIPTION
**Pull request by KevinGrandon**

### CODE REVIEW

File: src/browser.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+
 import React from 'react';
 import {createPlugin} from 'fusion-core';
 import {HelmetProvider} from 'react-helmet-async';
 
-export default __BROWSER__ &&
+import type {FusionPlugin} from 'fusion-core';
+
+const plugin =
+  __BROWSER__ &&
```
Comment (alxmyth) [inline src/browser.js:16]: Hmm, @ganemone - did you find that this broke treeshaking?

Comment (KevinGrandon) [reply src/browser.js:16]: Oh, thought this is what we did to fix treeshaking? Let's find the previous example of where it broke.

Comment (KevinGrandon) [reply src/browser.js:16]: I believe this was the tree-shaking fix you're referring to: https://github.com/fusionjs/fusion-plugin-rpc/commit/92c3a8ae4c53071bf31dfe1343c5901c75da7c6e

It looks like this is the correct way of doing things, and the incorrect way is:
```
export default ((__NODE__ && plugin: any): RPCPluginType);
```

Comment (alxmyth) [reply src/browser.js:16]: Got it, thanks for hunting that down!

**Review by alxmyth — APPROVED** [review]

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**